juntagrico_billing/templates/jb/user_bill.html

```diff
@@ -78,24 +78,29 @@ <h4>{% trans "Payments per" %} {{ today|date:"SHORT_DATE_FORMAT" }}:</h4>
         <div class="col-sm-2 text-right">{{ payment.amount | floatformat:2 }}</div>
     </div>    
     {% endfor %}
+    {% if open_amount != 0 %}
     <div class="row font-weight-bold mt-3">
-        {% if bill.paid %}
-        <div class="col">{% trans "Bill paid completely" %}</div>
-        {% else %}
         <div class="col-sm-5">{% trans "Amount open yet" %}:</div>
         <div class="col-sm-5"></div>
         <div class="col-sm-2 text-right">{{ open_amount | floatformat:2 }}</div>
         {% endif %}
```

Comment:
endif should be below `</div>`

Reply:
you are right, thanks!
made some more changes to so that the "Amount open" line is displayed too when there are no payments.